ibm_catalog.json

```diff
@@ -0,0 +1,163 @@
+      ],
+      "keywords": [
+        "vpc",
+        "slz",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution",
+        "vsi",
+        "virtual server instance"
+      ],
+      "short_description": "Creates a secure infrastructure with virtual servers to run your workloads on a VPC network",
+      "long_description": "The VSI on VPC landing zone provides secure and customizable compute resources for running your applications and services. It creates secure and compliant Virtual Server Instances (VSI) on top of an existing Virtual Private Cloud (VPC) network.",
+      "offering_icon_url": "https://globalcatalog.cloud.ibm.com/api/v1/1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc/artifacts/solution.svg",
+      "provider_name": "IBM",
+      "flavors": [
+        {
+          "label": "Standard",
+          "name": "standard",
+          "usage": "Add an example of the use of the module in the following code block.\n\nUse real values instead of \"var.\\u003cvar_name\\u003e\" or other placeholder values\nunless real values don't help users know what to change.\n--\\u003e",
+          "working_directory": "solutions/standard",
+          "licenses": [
+            {
+              "id": "LICENSE",
+              "name": "LICENSE",
+              "type": "text/plain",
+              "description": "LICENSE"
+            }
+          ],
+          "compliance": {},
+          "iam_permissions": [
+            {
+              "service_name": "is.vpc",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Operator"
+              ]
+            }
+          ],
+          "architecture": {
+            "descriptions": "",
+            "diagrams": {
+              "caption": "VSI on VPC landing zone - Standard variation",
+              "url": "https://raw.githubusercontent.com/terraform-ibm-zvsi/blob/standard_quickstart/reference-architecture/Standard-variation.svg",
+              "type": "image/svg+xml"
+            }
+          },
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key",
+              "type": "password",
+              "description": "The IBM Cloud platform API key needed to deploy IAM enabled resources.",
+              "required": true
+            },
+            {
+              "custom_config": {
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "type": "json_editor"
+              },
+              "key": "override_json_string"
+            },
+            {
+              "key": "prefix",
+              "type": "string",
+              "default_value": "__NOT_SET__",
+              "description": "A unique identifier for resources. Must begin with a lowercase letter and end with a lowerccase letter or number. This prefix will be prepended to any resources provisioned by this template. Prefixes must be 16 or fewer characters.",
+              "required": true
```

Comment:
No need to add `prefix` here - its already defined in your variables.tf file as a required and secure value so no need to duplicate it here. Only list variables here that you want to add extra customization to in catalog.

Reply:
same for quickstart flavour